pkg/importer/vddk-datasource_test.go

```diff
@@ -724,6 +724,86 @@ var _ = Describe("VDDK get block status", func() {
 			}
 		}
 	})
+	It("VDDK return more than 2000 changed block area", func() {
```

Comment:
You probably want to move this test under the other describe (`Describe("VDDK data source", func() {`), otherwise, you don't have the client creation mocked https://github.com/kubevirt/containerized-data-importer/blob/d24d618956524959f444d23ddf61796491454f97/pkg/importer/vddk-datasource_test.go#L53-L64

Reply:
Thanks fixed!